R/derive_advs_params.R

```diff
@@ -0,0 +1,632 @@
+#' Adds a Parameter for Waist to Hip Ratio
+#'
+#' @description Adds a record for Waist to Hip Ratio using Waist Circumference and Hip Circumference
```

Comment:
Minor: align language
```suggestion
#' @description Adds a parameter for Waist to Hip Ratio using Waist Circumference and Hip Circumference
```